packages/base/card-api.gts

```diff
@@ -1894,6 +1894,17 @@ export function subscribeToChanges(
     subscribers.set(fieldOrCard, changeSubscribers);
   }
   changeSubscribers.add(subscriber);
+
+  let fields = getFields(fieldOrCard, {
+    usedFieldsOnly: true,
+    includeComputeds: false,
+  });
+  Object.keys(fields).forEach((fieldName) => {
+    let value = peekAtField(fieldOrCard, fieldName);
+    if (isCardOrField(value)) {
+      subscribeToChanges(value, subscriber);
+    }
+  });
```

Comment:
this is good. one thing that occurred to me is that we probably want to protect against cycles. can we also add a guard at the beginning of this function that if `changeSubscribers.has(subscriber)` then we return early. that way we won't accidently get caught in a cycle.